src/components/account/CreateAccount.tsx

```diff
@@ -109,10 +110,8 @@ const CreateAccount = () => {
           } else {
             setFormValidationErrors(null)
             setRequestStatus('fetching')
-            fetch('https://nestjs-bank-app.herokuapp.com/accounts', {
-              method: 'POST',
-              headers: { 'Content-Type': 'application/json' },
-              body: JSON.stringify({
+            ky.post('https://nestjs-bank-app.herokuapp.com/accounts', {
```

Comment:
I think it's worth creating a ky object (`ky.create()`) in utils/ that takes the backend url as the `prefixUrl`. If/When this endpoint changes it's good to have it in one place. This will also show more obviously what path you're calling.
```suggestion
            ky.post('accounts', {
```